frontend/components/utils/FormatCurrency.vue

```diff
@@ -44,6 +43,15 @@ export default Vue.extend({
             }
             return this.value > 0 ? '+' : '';
         },
+        isValidNumber(): boolean {
+            if (BigNumber.isBigNumber(this.value)) {
+                if (this.value.toString() === 'NaN') {
+                    return false;
+                }
+            }
```

Comment:
Why not use existing method?

```suggestion
            if (BigNumber.isBigNumber(this.value) && this.value.isNaN()) {
                return false;
            }
```

https://mikemcl.github.io/bignumber.js/#isNaN

Also, where do you actually test this edge case?

Reply:
> Also, where do you actually test this edge case?

This was an issue that arose in `generateFakeAuction` resulting in failing tests and errors in the storyboard. It was never an issue before this, as BigNumber just displayed the missing BigNumber as `NaN`, however this new component threw an error when this kind of BigNumber was given. That is why I needed to add this check. 

I also added a Story, just to be aware of this edge case.